src/applications/facility-locator/containers/VAMap.jsx

```diff
@@ -169,6 +205,25 @@ class VAMap extends Component {
     this.listener();
   }
 
+  /**
+   * Helper method to compare search parameters between
+   * component updates/renders.
+   *
+   * Currently compares search string, location type,
+   * service type, and map bounding box.
+   *
+   * @param {object} previous Previous component props
+   * @param {object} current Current componet props
+   */
+  didParamsChange = (previous, current) => {
```

Comment:
Looks like the only time this is used is commented out for now. What do you think of also commenting this helper function as well?

Not sure how much it matters either way, but if that commented-out usage is removed without being implemented, then we're left with a function that doesn't ever get called, y'know? :man_shrugging:

Reply:
I can comment it out, sure. I left it uncommented as it's still a useful function and does work. Just not used yet based on the commented out section you mention.